source/user-manual/api/rbac/rbac-database-integrity.rst

```diff
@@ -0,0 +1,28 @@
+.. Copyright (C) 2021 Wazuh, Inc.
```

Comment:
Move this new file to development documentation, not here, I believe this is not user related documentation.

Reply:
done